### ISSUE TITLE
TypeError: Cannot read property 'returnValidation' of undefined

### ISSUE THREAD

**Issue by NameFILIP** [js-dxtools/webpack-validator#122]
After updating from 2.2.3 to 2.2.5 started getting this error:

```
TypeError: Cannot read property 'returnValidation' of undefined
    at validate (/Users/.../node_modules/webpack-validator/dist/index.js:122:29)
    ...
```

In my webpack.config.js I have:

```
const webpackConfig = { ... };
module.exports = validate(webpackConfig);
```

The workaround is to pass an empty object as a second argument:
```
module.exports = validate(webpackConfig, {});
```
but it looks like a regression and should be fixed

**Comment by oieduardorabelo** [comment]
Yep, here too, 2.2.3 to 2.2.5

**Comment by kentcdodds** [comment]
I think #121 will fix this.

**Comment by kentcdodds** [comment]
This will be released automatically as 2.2.6 soon

**Comment by kentcdodds** [comment]
Please re-open this issue if it is not resolved.
